src/openzaak/components/documenten/api/fields.py

```diff
@@ -27,6 +27,10 @@ def to_representation(self, value):
             return super().to_representation(value)
 
         value = value.latest_version
+
+        if value is None:
```

Comment:
Why are we returning the super().to_representation() without the value?

If there is no good reason then I would suggest to return the super at the end of the method, and have a check to see if we want to return the reverse of "enkelvoudiginformatieobject-detail".

Reply:
Idk, it probably makes more sense to just return an empty string or None, but this situation is not supposed to happen in the first place.

Reply:
If it isn't supposed to happen shouldn't we raise an exception instead or atleast log it? I might be thinking too much into it though.